charts/rancher-eks-operator/1.0.700/values.yaml

```diff
@@ -0,0 +1,11 @@
+global:
+  systemDefaultRegistry: ""
+
+eksOperator:
+  image:
+    repository: rancher/eks-operator
+    tag: v1.0.6
```

Comment:
Did nothing else make it into eks-operator since 2.5.6 besides this ?

Reply:
Correct, only the fix for encoded user data made it in.